kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/enums/KafkaClientType.java

```diff
@@ -7,7 +7,7 @@
 package io.kroxylicious.systemtests.enums;
 
 public enum KafkaClientType {
-    JAVA,
+    CLI,
```

Comment:
They are all CLIs so CLI doesn't really help.  Maybe `STRIMZI_TEST_CLIENT`.

Reply:
I'm short on better ideas, not a huge fan of `STRIMZI_TEST_CLIENT` as I feel like it implies we are testing Strimzi or with Strimzi specific tools but its better than any thing else I can come up with.

Reply:
Done. Thanks for the naming!